packages/ember-application/lib/system/engine-instance.js

```diff
@@ -220,13 +219,12 @@ EngineInstance.reopenClass({
     registry.injection('view', '_environment', '-environment:main');
     registry.injection('route', '_environment', '-environment:main');
 
+    let rendererType = 'inert';
     if (options.isInteractive) {
-      registry.injection('view', 'renderer', 'renderer:-dom');
-      registry.injection('component', 'renderer', 'renderer:-dom');
-    } else {
-      registry.injection('view', 'renderer', 'renderer:-inert');
-      registry.injection('component', 'renderer', 'renderer:-inert');
+      rendererType = 'dom';
```

Comment:
DRY doesn't mean reduce all duplication at all costs, it should be about duplication of units of knowledge in a codebase. Having a couple of simple method calls repeated is fine, it's very clear what it does.